src/toolbox/shamir.rs

```diff
@@ -240,27 +250,28 @@ impl SecretShare {
     }
 }
 
+#[allow(unused_imports)]
 mod tests {
```

Comment:
Nice testing! I would add some cases for threshold 1, as mentioned above

Reply:
Done!